adoc/admin-migrate-caasp4.5.adoc

```diff
@@ -1,11 +1,12 @@
 == Migration to {productname} 4.5
 
-1) The OS must be connected to SCC. Otherwise the migration will not work.
-In order to reconnect your OS to SCC, you run:
+1) The OS must be connected to servers like SCC or RMT. Otherwise the migration will not work.
```

Comment:
The server must be ...

Reply:
Scratch "Otherwise the migration will not work." and replace it with a reference to the SLES documentation. Like "Other migration scenarios are covered in the SLES upgrade guide"